plugins/rapid7_insightvm/unit_test/test_validate_user.py

```diff
@@ -62,5 +62,8 @@ def test_validate_role_exists(self):
         session = requests.session()
         test_object = resource_helpers.ValidateUser(logger=logger, session=session)
         test_object.validate_user_email(user["email"])
-        with pytest.raises(PluginException, match="The email address for user account was not valid!"):
+        with pytest.raises(
+            PluginException,
+            match="An error occurred during plugin execution!\n\nThe email address for user account was not valid Ensure that the email address is correct",
+        ):
```

Comment:
Maybe you should use with self.assertRaises instead of pytest.raises to be consistent with other unit_tests.
Like in the test above:
```
        with self.assertRaises(PluginException) as e:
            self.action.run({Input.DATE: date, Input.ID: exception_id})
        self.assertEqual(e.exception.cause, cause)
        self.assertEqual(e.exception.assistance, assistance)
        self.assertEqual(e.exception.data, data)
```